tests/bdd/bdd_test.go

```diff
@@ -0,0 +1,247 @@
+package bdd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	v1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis"
+	chaosEngineV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	clientV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned/typed/litmuschaos/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	scheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var kubeconfig = string(os.Getenv("HOME") + "/.kube/config")
+var config, _ = clientcmd.BuildConfigFromFlags("", kubeconfig)
+var client, _ = kubernetes.NewForConfig(config)
+var clientSet, _ = clientV1alpha1.NewForConfig(config)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+
+	err := v1alpha1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//Creating crds
+	By("creating chaosengine crd")
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/crds/chaosengine_crd.yaml").Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	By("creating chaosexperiment crd")
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/crds/chaosexperiment_crd.yaml").Run()
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	By("creating chaosresult crd")
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/crds/chaosresults_crd.yaml").Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//creating rbacs
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/rbac.yaml").Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//creating opearator
+
+	By("creating operator")
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/operator.yaml").Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Println("operator created successfully")
```

Comment:
Instead of `fmt` can you use the `log` package which is already using in this repo